config/secrets.yml

```diff
@@ -11,10 +11,10 @@
 # if you're sharing your code publicly.
 
 development:
-  secret_key_base: 8f85b60688ca164debb7960ac838f074dbb54def7299ee7f2363afe5da783d5595b741dd2c2c17d1ee5f3bc6de18dc540240fc8064525ba1c157ce604143c8db
+  secret_key_base: dc889de5ddd86a8f57ec040472800e3703c425d70eaf3721cbdc7e759e9b1125abb6bb45c073dd1c3045ee036c2201a37803804085308c4c6671e11ebbe6a9a5
 
 test:
-  secret_key_base: ce60ba18c3631123069bd459b0b2e755effdf5be8017786760cc063046b3bbf318515ed5fa1204aab7a98be14c3a6b9ef03aaa9c03658648fe6818de7d1e9281
+  secret_key_base: dc889de5ddd86a8f57ec040472800e3703c425d70eaf3721cbdc7e759e9b1125abb6bb45c073dd1c3045ee036c2201a37803804085308c4c6671e11ebbe6a9a5
 
 # Do not keep production secrets in the repository,
```

Comment:
Shouldn't this file be in the gitignore?

Reply:
/shrug

It wasn't there to start with so I didn't mess with it.

Reply:
100% should be